package.json

```diff
@@ -32,7 +33,7 @@
     "prettier": "^2.2.1",
     "ts-jest": "^27.0.5",
     "tsukuru": "^0.7.2",
-    "typescript": "~4.1.3"
```

Comment:
This caused peer dependencies incompatibility on npm 8/9

Reply:
Please keep merge requests focused on their topic. I updated the packages properly.